app/src/main/AndroidManifest.xml

```diff
@@ -35,7 +35,8 @@
       android:theme="@style/OppiaThemeWithoutActionBar" />
     <activity
       android:name=".app.help.HelpActivity"
-      android:theme="@style/OppiaThemeWithoutActionBar" />
+      android:theme="@style/OppiaThemeWithoutActionBar"
+      android:label="@string/menu_help" />
```

Comment:
Create a new string `help_activity_title ` and use it here.

Reply:
created new string named help_activity_title